gateway/Dockerfile

```diff
@@ -32,12 +40,14 @@ COPY version        version
 COPY scaling        scaling
 COPY pkg            pkg
 COPY main.go        .
+
+
 RUN license-check -path ./ --verbose=false "Alex Ellis" "OpenFaaS Authors" "OpenFaaS Author(s)"
 
 
 # Run a gofmt and exclude all vendored code.
 RUN test -z "$(gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*"))"
-RUN go test $(go list ./... | grep -v integration | grep -v /vendor/ | grep -v /template/) -cover
+# RUN go test $(go list ./... | grep -v integration | grep -v /vendor/ | grep -v /template/) -cover
```

Comment:
Is this supposed to be commented out? if so, can we just remove it

Reply:
No, the unit tests are required. Let me fix that.